pkgs/by-name/wr/wrangler/package.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  llvmPackages,
+  musl,
+  xorg,
+  stdenv,
+  fetchFromGitHub,
+  pnpm_9,
+  nodejs,
+  makeWrapper,
+  autoPatchelfHook,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wrangler";
+  version = "3.62.0";
+
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = "workers-sdk";
+    rev = "wrangler@${finalAttrs.version}";
+    hash = "sha256-/4iIkvSn85fkRggmIha2kRlW0MEwvzy0ZAmIb8+LpZQ=";
+  };
+
+  buildInputs = [
+    llvmPackages.libcxx
+    llvmPackages.libunwind
+    musl
+    xorg.libX11
+  ] ++ lib.optional stdenv.isLinux autoPatchelfHook;
+
+  nativeBuildInputs = [
+    nodejs
+    pnpm_9.configHook
+    makeWrapper
+    autoPatchelfHook
```

Comment:
It'd be nice to put a comment explaining this. IIRC there was a comment here in one of the revisions of the PR.

P.S The issue with that dependency which requires `patchelf` is also causing the build to fail on platforms other then `x86_64-linux` - which warrants a `meta.platforms` line. Please mention that in the comment here, or near the `meta.platformts` line.